runtime/onert/core/include/backend/ITensor.h

```diff
@@ -103,6 +103,17 @@ class ITensor
    * @note  Higer dimension will be placed on front.
    */
   virtual ir::Shape getShape() const;
+
+  virtual bool is_subtensor() const { return false; }
+  virtual bool hasClBuffer() const { return false; }
+  virtual void enqueueWriteBuffer(const void *)
```

Comment:
Shouldn't it better for ITensor to have names neutral to specific backend?
Or how about something like the following?

- hasClBuffer -> needMemoryMap: // some backend needs CPU memory to be mapped to, e.g., GPU memory or NPU memory.
- enqueueWriteBuffer -> mapMemory

Reply:
> - hasClBuffer -> needMemoryMap: // some backend needs CPU memory to be mapped to, e.g., GPU memory or NPU memory.

Okay. I will.

> - enqueueWriteBuffer -> mapMemory

This method is not to map memory. How about changing it to writeBuffer?

Reply:
how about writeMappedMemory ?

Reply:
This method is not related to mapping data. It writes a buffer to device memory of the backend tensor from host memory.

Reply:
I see. The concern I have is that if the name is _too general_ like `writeBuffer`, backend developer may be curious why CPU tensor cannot support `writeBuffer`. In that sense I like `enqueueWriteBuffer` (your first naming) would be better than `writeBuffer`. Or some long name like `writeBufferToDevice`(?) would also work.

Reply:
I think the original(enqueueWriteBuffer) looks the best among them.